cmd/nerdctl/system_prune.go

```diff
@@ -103,7 +104,11 @@ func systemPruneAction(cmd *cobra.Command, args []string) error {
 	if err := containerPrune(ctx, cmd, client, globalOptions); err != nil {
 		return err
 	}
-	if err := networkPrune(ctx, cmd, client, globalOptions); err != nil {
+	if err := network.Prune(ctx, types.NetworkPruneCommandOptions{
+		GOptions:             globalOptions,
+		Force:                force,
+		NetworkDriversToKeep: networkDriversToKeep,
+	}, cmd.InOrStdin(), cmd.OutOrStdout()); err != nil {
```

Comment:
related to my previous comments

> can you keep this logic in cmd side (networkPruneAction)? I feel this kind of confirmation logic is only used by cmd and I don't think others need to reuse this logic.

If we don't change, here the confirmation will show up again right? (the user has confirmed at line 77 here)